## 50_mysql-test__db_test.dpatch by Christian Hammers <ch@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Patch 33_scripts__mysql_create_system_tables__no_test removes the
## DP: rights for anybody to connect to the test database but the test
## DP: suite depends on them.
Index: mariadb-10.0/mysql-test/mysql-test-run.pl
===================================================================
--- mariadb-10.0.orig/mysql-test/mysql-test-run.pl	2015-08-13 19:40:49.125504433 +0900
+++ mariadb-10.0/mysql-test/mysql-test-run.pl	2015-08-13 19:41:10.221615248 +0900
@@ -3605,6 +3605,11 @@
     mtr_appendfile_to_file("$sql_dir/mysql_system_tables_data.sql",
 			   $bootstrap_sql_file);
 
+    mtr_tofile($bootstrap_sql_file, "-- Debian removed the default privileges on the 'test' database\n");
+    mtr_tofile($bootstrap_sql_file, "INSERT INTO mysql.db VALUES ('%','test','','Y','Y','Y','Y','Y','Y','N','Y','Y','Y','Y','Y','Y','Y','Y','N','N','Y','Y');\n");
+    mtr_tofile($bootstrap_sql_file, "INSERT INTO mysql.db VALUES ('%','test\\_%','','Y','Y','Y','Y','Y','Y','N','Y','Y','Y','Y','Y','Y','Y','Y','N','N','Y','Y');\n");
+
+
     # Add test data for timezone - this is just a subset, on a real
     # system these tables will be populated either by mysql_tzinfo_to_sql
     # or by downloading the timezone table package from our website
